.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2020: true,
```

Comment:
Instead of disabling eslint's no-undef rule in your test file to allow undeclared variables you could just add jest to your eslint config.